src/QsCompiler/Tests.Compiler/TestCases/Optimizer/LoopUnrolling_output.txt

```diff
@@ -1,22 +1,9 @@
 namespace Microsoft.Quantum.Testing.Optimization.LoopUnrolling {
-    Operation Test () : UnitType {
+    Operation Test () : Int {
         QsBody () {
-            {
-                {
-                    let i = IntLiteral 0L;
-                    { EmptyScope }
-                }
-                {
-                    let i = IntLiteral 2L;
-                    {
-                        let r = IntLiteral 3L;
-                    }
-                }
-                {
-                    let i = IntLiteral 4L;
-                    { EmptyScope }
-                }
-            }
+            mutable r = 0;
+            set LocalVariable r = 3;
```

Comment:
Nice!

Reply:
It would be good to also have some test cases where no unrolling is/can be done.